examples/fabric-admin/commands/pairing/PairingCommand.cpp

```diff
@@ -564,15 +586,46 @@ void PairingCommand::OnCurrentFabricRemove(void * context, NodeId nodeId, CHIP_E
 
 chip::Optional<uint16_t> PairingCommand::FailSafeExpiryTimeoutSecs() const
 {
-    // We don't need to set additional failsafe timeout as we don't ask the final user if he wants to continue
+    // No manual input, so do not need to extend.
     return chip::Optional<uint16_t>();
 }
 
+bool PairingCommand::ShouldWaitAfterDeviceAttestation()
+{
+    // If there is a vendor ID and product ID, request OnDeviceAttestationCompleted().
+    SetupPayload payload;
+    CHIP_ERROR err = GetPayload(mOnboardingPayload, payload);
+    return err == CHIP_NO_ERROR && payload.vendorID != 0 && payload.productID != 0;
```

Comment:
Why don't we validate if the example app only sets either the vendorID or productID, but not both?

Reply:
If anything these should be checked when RequestCommissioningApproval is received. But, I could also add a case below that fails if one is set and not the other. Thoughts?

Reply:
I added the check here as well. Closing as part of this PR's scope.